app/models/survey.rb

```diff
@@ -1,8 +1,42 @@
+require 'will_paginate/array'
+require 'date'
+
 class Survey < ActiveRecord::Base
+  include Trackable
   belongs_to :user
   has_many :questions
   belongs_to :company
 
   validates :name, presence: true, length: { maximum: 20 }
   validates :description, presence: true, length: { maximum: 150 }
+
+  scope :public_surveys, -> { where(type: 'public') }
+  scope :expired_surveys, ->(date_time) { where('expiry < ?', date_time) }
```

Comment:
why to get date_time when you know which surveys are expired

Reply:
resolved